CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+
+- `opentelemetry-instrumentation-aws-lambda` Added additional consumers. Fixes issue 902  
```

Comment:
I don't think we normally include the issue number in the CHANGELOG? The PR description will have it. Probably best to remove in this case.

```suggestion
- `opentelemetry-instrumentation-aws-lambda` `SpanKind.SERVER` by default,  add more cases for `SpanKind.CONSUMER` services
```